server/protocols/jmap/src/test/java/org/apache/james/jmap/utils/JsoupHtmlTextExtractorTest.java

```diff
@@ -24,17 +24,16 @@
 import java.nio.charset.StandardCharsets;
 
 import org.apache.commons.io.IOUtils;
-import org.apache.james.mailbox.tika.extractor.TikaTextExtractor;
 import org.junit.Before;
 import org.junit.Test;
 
-public class MailboxBasedHtmlTextExtractorTest {
+public class JsoupHtmlTextExtractorTest {
 
-    private MailboxBasedHtmlTextExtractor textExtractor;
+    private JsoupHtmlTextExtractor textExtractor;
```

Comment:
In an ideal word the JsoupHtmlTextExtractorTest should be introduced in the commit where JsoupHtmlTextExtractor is introduced.
Therefore I would add it in the commit before and remove MailboxBasedHtmlTextExtractorTest in this commit